build/check.js

```diff
@@ -109,6 +109,34 @@ gulp.task('check-javascript-format', function() {
       .pipe(gulpClangFormat.checkFormat('file', undefined, {verbose: true, fail: true}));
 });
 
+/**
+ * Checks whether project's Go files are formatted by goimorts.
+ */
+gulp.task('check-go-format', function(doneFn) {
+  function execDoneFn(stdout, stderr) {
+    if (stdout instanceof Error) {
+      console.log(stderr);
+      doneFn(stdout);
+    } else {
+      if (stdout.length) {
+        console.log(stdout);
+        doneFn(Error(
+            'Above files have unformatted codes. ' +
```

Comment:
The above files have unformatted code.

Reply:
done

Reply:
code is singular (uncountable)

'The above files have unformatted code.'